neo/public/javascripts/timeseries/controllers.js

```diff
@@ -7,6 +7,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
     $scope.dc = $window.dc;
     $scope.crossfilter = $window.crossfilter;
     $scope.empty = [];
+    $scope.responsiveWidth = $(window).width();

```

Comment:
This name was intended for the responsive design: meaning the width can adjust itself according to the resizing event. I will change it now as we don't need that much responsiveness.